overlord/devicestate/devicemgr.go

```diff
@@ -60,9 +60,15 @@ var (
 // policies.
 type DeviceManager struct {
 	systemMode string
+	// saveAvailable keeps track whether /var/lib/snapd/save
+	// is available, i.e. exists and is mounted from ubuntu-save
+	// if the latter exists.
+	// TODO: it must be set to false if ubuntu-save is unmounted.
```

Comment:
yes, but probably it will need changes, it's mostly a placeholder for the real implementation, but given that the real implementation will touch many places it seems better to have at least a flag already to find them, instead of a family of TODO that are hard to relate. It also already make sure that we don't try to use save on UC18/16 by mistake